testdata/scripts/scopegen_no_swagger_v2.txt

```diff
@@ -0,0 +1,43 @@
+gunk generate -v echo.gunk
+exec go build all.scopes.go
+cmp all.scopes.json all.scopes.json.golden
+cmp all.scopes.go all.scopes.go.golden
+-- go.mod --
+module testdata.tld/util
+
+require (
+	github.com/gunk/opt v0.0.0-20190514110406-385321f21939
+)
+-- .gunkconfig --
+[generate]
+command=scopegen
+go=true
+json=true
+output_version=2
+-- echo.gunk --
+package test
+
+import (
+	"github.com/gunk/opt/http"
+	"github.com/gunk/opt/openapiv2"
+)
+
+// Message comment
+type Message struct {
+	Name string `pb:"1" json:"name"`
+}
+
+type Service interface {
+	GetMessage(Message) Message
+}
+-- all.scopes.json.golden --
```

Comment:
out of scope for the PR but why do we generate the scope files when the definitions don't contain scopes?

Reply:
It is clearer/more convenience to output an empty scopegen file than omit it. No file generated could mean many things, an empty file clearly told user that the file has successfully generated.